src/bp/sdk/botpress.d.ts

```diff
@@ -37,6 +37,7 @@ declare module 'botpress/sdk' {
     id?: number
     password?: string
     salt?: string
+    tokenVersion?: number
```

Comment:
Why is tokenVersion optional here? I don't see any case where it can be. As mush when I look at the strategy table creation class (StrategyUserTable) as when I look at the migration.